sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -13,6 +13,7 @@ def generate_solr_document
         # between files on disk (in fcrepo.binary-store-path) and objects
         # in the repository.
         solr_doc[Solrizer.solr_name('digest', :symbol)] = digest_from_content
+        object.index_collection_ids(solr_doc)
```

Comment:
I originally did not do this unless config.collection_facet had a value other than nil.  I can return to that.  What it buys you if it is always indexed is you can change from nil to :public or :user and the information for the facets is already there.  But since @jcoyne pointed out that this method is expensive, maybe it should only be there when facets are being used.